apps/fortran/airfoil/airfoil_arg_ptrs/dp/input.F90

```diff
@@ -33,26 +33,34 @@ end subroutine getSetSizes
 
 
 ! fill up arrays from file
-subroutine getSetInfo ( nnode, ncell, nedge, nbedge, cell, edge, ecell, bedge, becell, bound, x, q, qold, res, adt )
```

Comment:
Why not pass the arrays here?

Reply:
ok, can be fixed. Already done in the next commit.